tests/unit-tests/platforms/x11/test_platform.cpp

```diff
@@ -88,6 +89,7 @@ TEST_F(X11GraphicsPlatformTest, failure_to_open_x11_display_results_in_an_error)
         {
             std::make_shared<mg::X::Platform>(
                 nullptr,
+                std::string("Mir on X"),
```

Comment:
Since its introduction in C++11 we tend to use uniform initialization syntax (braces for initialization) everywhere.

But here there is no need as there is a converting constructor.

```suggestion
            "Mir on X",
```